org.gtk.Gtk3theme.Orchis.yml

```diff
@@ -0,0 +1,38 @@
+id: org.gtk.Gtk3theme.Orchis
+branch: "3.22"
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+
+build-extension: true
+appstream-compose: false
+separate-locales: false
+
+modules:
+  - name: Orchis
+    buildsystem: simple
+    build-commands:
+      - install -dm755 /usr/share/runtime/share/themes
+      - mkdir -p dist
+      - tar xvf release/Orchis.tar.xz -C dist/
+      - cp -rv dist/Orchis /usr/share/runtime/share/themes
+    sources:
+      - type: archive
+        url: https://api.github.com/repos/vinceliuice/Orchis-theme/tarball/2024-04-18
+        dest-filename: Orchis.tar.gz
+        sha256: 7f4fab7e19d0f372ee5985333bf4c46884814631bad2b20dcf8438cda0753b11
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/vinceliuice/Orchis-theme/releases/latest
+          tag-query: .tag_name
+          version-query: "$tag"
+          url-query: .tarball_url
+
+  - name: appdata
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 ${FLATPAK_ID}.appdata.xml -t ${FLATPAK_DEST}/share/appdata
+      - appstream-compose --basename=${FLATPAK_ID} --prefix=${FLATPAK_DEST} --origin=flatpak ${FLATPAK_ID}
```

Comment:
this is no longer necessary. Also delete line 8.

Reply:
I removed it.